src/web/services/notification.service.ts

```diff
@@ -28,4 +28,14 @@ export class NotificationService {
   getNotifications(): Observable<Notifications> {
     return this.httpRequestService.get(ResourceEndpoints.NOTIFICATION);
   }
+
+  /**
+   * Retrieve all notifications for a specific target user type.
```

Comment:
```suggestion
   * Retrieves all notifications for a specific target user type.
```
Minor nitpick. Might want to fix the grammar for line 26 in the same file as well.